flink-streaming-java/src/main/java/org/apache/flink/streaming/util/latency/LatencyHistogram.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.util.latency;
+
+import org.apache.flink.metrics.Histogram;
+import org.apache.flink.metrics.HistogramStatistics;
+
+import org.apache.commons.math3.stat.descriptive.DescriptiveStatistics;
+
+/**
+ * The {@link LatencyHistogram} use a DescriptiveStatistics {@link DescriptiveStatistics} as a Flink {@link Histogram}.
+ */
+public class LatencyHistogram implements org.apache.flink.metrics.Histogram {
```

Comment:
This class has actually little to do with latency. It's just a wrapper for a DescriptiveStatistics that we happen to use for measuring latency, as such i would rename the class/fields, and move it out of the latency namespace.

Reply:
I do not know how to change, could you give some suggestions?

Reply:
For the name i would suggest `DescriptiveStatisticsHistogram`. As for the location, `org.apache.flink.runtime.metrics` in flink-runtime, the same place where the `MeterView` class is located.

Reply:
make  sense.
But, the  `org.apache.flink.streaming.runtime.streamrecord.LatencyMarker` in flink-streaming-java module, if `DescriptiveStatisticsHistogram`  moved to flink-runtime, need add dependency flink-streaming-java for flink-runtime pom.xml.

Reply:
Only the Histogram and HistogramStatistics should be moved to flink-runtime. They both do not require the LatencyMarker. The `LatencyStats` class remains in flink-streaming-java.